### DISCUSSION TITLE
Desire for RC releases?

### DISCUSSION THREAD

**Discussion by p-mongo** [mongodb/mongoid#5072] [General]
Hi folks,

I am wondering if there is any interest in the Ruby team performing release candidate releases for bson-ruby, the driver and Mongoid?

We have been doing it primarily in 2019-2020, and while we have received some feedback (i.e. bug reports) on the RCs, I personally feel that the RCs aren't used much. Most recently, Mongoid 7.3.0 was released without any release candidates.

Do you find RCs of Mongoid, driver and bson-ruby valuable?

**Comment by johnnyshields** [comment]
I will be happy to use RCs in production and give feedback. At the moment I am just using direct Git commits hashes of each project but I feel a little uneasy about this as it's not 100% clear to me whether this is "safe" or if there are known issues. So if there can be RC tags where the Mongo team is effectively indicating "We think this is ready, we aren't aware of known issues" it would be a big help for me.

**Comment by p-mongo** [comment]
That is great to hear Johnny. Are you watching the repos to be notified of the releases?
